cli/cmd/digger/root.go

```diff
@@ -90,9 +89,9 @@ func PreRun(cmd *cobra.Command, args []string) {
 
 	hostName := os.Getenv("DIGGER_HOSTNAME")
 	token := os.Getenv("DIGGER_TOKEN")
-	orgName := os.Getenv("DIGGER_ORGANISATION")
+	//orgName := os.Getenv("DIGGER_ORGANISATION")
 	BackendApi = backend.NewBackendApi(hostName, token)
-	PolicyChecker = policy.NewPolicyChecker(hostName, orgName, token)
+	//PolicyChecker = policy.NewPolicyChecker(hostName, orgName, token)
```

Comment:
Commenting this out breaks the policy checker

```19:29:25 github.go:342: Following projects are impacted by pull request #152
19:29:25 github.go:344: - root
19:29:25 github.go:346: GitHub event processed successfully
19:29:25 github.go:378: GitHub event converted to commands successfully
19:29:25 github.go:439: Following commands are going to be executed:
project: root: commands: "digger plan", 
19:29:25 policy.go:272: Error while fetching policy: unexpected response while fetching project policy: Not allowed to access this resource with this access level code 403
19:29:25 usage.go:112: Failed to run commands. error checking policy: unexpected response while fetching project policy: Not allowed to access this resource with this access level code 403```

Reply:
Hey! We have moved the logic of fetching policy checkers to providers: https://github.com/diggerhq/digger/pull/1588/files#diff-476bae6fdca0f01c3e01eb49e676ec25f2de056f623d578aa8eb03b35940376bR12

Thats why it has been commented out from here , because its passed as a provider into the function: https://github.com/diggerhq/digger/pull/1588/files#diff-df1410c7e541e1f284e5260de3e777131a5ca5283fa4b8f36a03dfc5de9b2f29R27